presto-docs/src/main/sphinx/release/release-0.212.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.212
+=============
+
+General Changes
+---------------
+
+* Add the :func:`ST_LineString` function to form a ``LineString`` from an array of points.
+* Enhanced :func:`ST_ConvexHull` and :func:`convex_hull_agg` functions to support geometry collections.
+* Improve performance for some query plans that include ``MarkDistinct`` or ``DistinctLimit``.
+* Remove ``round(x, d)`` and ``truncate(x, d)`` functions where ``d`` is a ``BIGINT`` (:issue:`11462`).
+* Change filtered global aggregations to include an explicit filter operator.
+* Fix :func:`ST_GeomFromBinary` function when applied to more than one row.
```

Comment:
It's not clear what this means. How do you apply a function to multiple rows? Does this mean the function didn't work when the query processed multiple rows?

Also, is it fixing a failure? Incorrect result?